README.md

```diff
@@ -1,32 +1,32 @@
 # pelion-edge-provisioner (pep)
 
-It provisions the gateway for Pelion Edge solution to successfully and securely communicate with cloud. This tool hosts REST APIs and mongo database which helps create new and also store gateway-identites. The tool is designed to run on the factory floor and it provisions the gateways in production mode.
+The Pelion Edge provisioning tool provisions the gateway, so Pelion Edge can successfully and securely communicate with the cloud. This tool hosts REST APIs and a mongo database, which helps create and store gateway identities. The tool is designed to run on the factory floor, and it provisions the gateways in production mode.
 
 <img src="./assets/pep-arch.png" width="650" height="550"/>
 
 ## Prerequisite
 
-This tools wraps [Factory Configurator Utility (FCU)](https://www.pelion.com/docs/device-management/current/provisioning-process/index.html). Thus in order to install this tool, following files are required -
+This tools wraps [Factory Configurator Utility (FCU)](https://www.pelion.com/docs/device-management/current/provisioning-process/index.html). To install this tool, the following files are required:
```

Comment:
Are the files required in this order? Or could I install them in a different order?

Reply:
This is the recommended order.